src/compiler/emitter.ts

```diff
@@ -2777,24 +2805,45 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
                 }
             }
 
+            function emitAliasEqual(name: Identifier): boolean {
+                for (const specifier of exportSpecifiers[name.text]) {
+                    emitStart(specifier.name);
+                    emitContainingModuleName(specifier);
+                    write(".");

```

Comment:
Oh right, of course the identifier itself can be "default", didn't think of that. Thanks